### PR TITLE
Removed molecules components

### LINKED ISSUE #1684: Remove molecules from here
We have moved all molecules to neeto-molecules and have updated all products. Now we can delete those components from here.
Reference: https://github.com/bigbinary/neeto-molecules/issues/149

@KrishnapriyaSkk _a please work on this.

### PR DESCRIPTION
**Pull request by KrishnapriyaSkk**
- Fixes #1684

**Description**
Removed following components from `neetoui`:
- `Header`
- `SubHeader`
- `MenuBar`
- `SideBar`
- `PageLoader`
- `AppSwitcher`
- `Container`
- `Scrollable`
- `Page`

**Checklist**

- ~~[ ] I have made corresponding changes to the documentation.~~
- ~~[ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by KrishnapriyaSkk** [comment]
@Amaljith-K _a
Please review these changes.

File: src/components/layouts/Sidebar/HelpSection/Tooltip.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ const HelpSectionTooltip = ({ helpSectionProps }) => {
     },
   ].filter(Boolean);
 
+  // eslint-disable-next-line react/jsx-filename-extension
```
Comment (Amaljith-K) [inline src/components/layouts/Sidebar/HelpSection/Tooltip.js:44]: Why not make this file JSX instead of disabling eslint?

File: src/components/layouts/Sidebar/index.jsx [hunk 2]
```diff
@@ -129,13 +129,15 @@ Sidebar.propTypes = {
   /**
    * The name of the app that is currently active
    */
+  // eslint-disable-next-line react/no-unused-prop-types
   appName: PropTypes.string,
```
Comment (Amaljith-K) [inline src/components/layouts/Sidebar/index.jsx:133]: Why is this propType kept if we aren't using it anywhere?

**Review by Amaljith-K — COMMENTED** [review]
@KrishnapriyaSkk _a recheck the eslint-disable comments & assign back to me.

**Comment by KrishnapriyaSkk** [comment]
@Amaljith-K _a
I have removed the eslint-disable comments from a few places.

**Review by Amaljith-K — APPROVED** [review]
@amaldinesh7 _a this PR is going to release neetoui 5 with several components removed.
Please check and merge if everything looks alright to you.

File: package.json [hunk 3]
```diff
@@ -67,6 +67,8 @@
     "@bigbinary/eslint-plugin-neeto": "^1.0.26",
     "@bigbinary/neeto-commons-frontend": "^2.0.42",
     "@bigbinary/neeto-icons": "^1.9.22",
+    "@bigbinary/neeto-molecules": "^1.0.20",
+    "@bigbinary/neetoui": "^4.4.26",
```
Comment (amaldinesh7) [inline package.json:71]: Why is this added? @sruthy-j-mallya

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@KrishnapriyaSkk  _a Please check the comment.

File: package.json (hunk 3 above)
Comment (KrishnapriyaSkk) [reply package.json:71]: This was added to import `Header` and `SubHeader` components in `Scrollable.stories.jsx`, but then that component was removed in the further commits and this was accidentally left out.

Comment (KrishnapriyaSkk) [reply package.json:71]: Will remove those.

**Comment by KrishnapriyaSkk** [comment]
@amaldinesh7 _a
I have made the change. Please check.

File: package.json [hunk 4]
```diff
@@ -67,6 +67,7 @@
     "@bigbinary/eslint-plugin-neeto": "^1.0.26",
     "@bigbinary/neeto-commons-frontend": "^2.0.42",
     "@bigbinary/neeto-icons": "^1.9.22",
+    "@bigbinary/neeto-molecules": "^1.0.20",
```
Comment (amaldinesh7) [inline package.json:70]: Why do we need neeto-molecules here? @KrishnapriyaSkk _A

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (KrishnapriyaSkk) [reply package.json:70]: This was added to display the removed components in one of the removed stories, as discussed, I will remove that particular Table story inside layouts.

**Comment by KrishnapriyaSkk** [comment]
@amaldinesh7 _a
I have removed that particular story and neeto-molecules. Please check.
